tron/utils/scribereader.py

```diff
@@ -77,7 +77,7 @@ def read_log_stream_for_action_run(
     min_date: Optional[datetime.datetime],
     max_date: Optional[datetime.datetime],
     paasta_cluster: Optional[str],
-    max_lines: Optional[int] = None,
```

Comment:
it might be a good idea to find out what the average log length is so that we can pick a number that hopefully won't impact folks + make sure that we're in general keeping that number in mind as a target for what we should aim to support at a minimum

we can probably figure this out through science in a couple ways:
* check the log streams for all the jobs that are using k8s in other clusters and see what the average lines per run is by summing things up
* look at the logs on-disk on the last mesos-using cluster and see how big those files are (although, the functionality here is pretty brittle so a lot of these might be empty ;_;)
* see what the thresholds we used for cleaning/truncating the on-disk logs are and extrapolate from that to how many lines we'd like to keep

it might also be a good idea to have this default be configurable without having to re-deploy tron: perhaps we can use srv-configs here (we'll have to use slightly  different tooling than we do internally - the srv-configs library is not open-source so we'll need to use staticconf directly)?